src/main/java/kitchenpos/order/application/OrderService.java

```diff
@@ -0,0 +1,45 @@
+package kitchenpos.order.application;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import kitchenpos.order.domain.Order;
+import kitchenpos.order.domain.OrderRepository;
+import kitchenpos.order.domain.OrderStatus;
+import kitchenpos.order.dto.OrderRequest;
+import kitchenpos.order.dto.OrderResponse;
+import kitchenpos.order.event.CreateOrderEvent;
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+public class OrderService {
+    private final OrderRepository orderRepository;
+    private final ApplicationEventPublisher eventPublisher;
+
+    public OrderService(OrderRepository orderRepository, ApplicationEventPublisher eventPublisher) {
+        this.orderRepository = orderRepository;
+        this.eventPublisher = eventPublisher;
+    }
+
+    @Transactional
+    public OrderResponse create(final OrderRequest request) {
+        final Order savedOrder = orderRepository.save(request.toOrder());
+        eventPublisher.publishEvent(new CreateOrderEvent(request.getOrderTableId(), request.getMenuIds()));
+        return OrderResponse.from(savedOrder);
+    }
+
+
+    @Transactional(readOnly = true)
+    public List<OrderResponse> list() {
+        return OrderResponse.from(orderRepository.findAll());
+    }
+
+    @Transactional
+    public OrderResponse changeOrderStatus(final Long orderId, final OrderRequest request) {
+        final Order findOrder = orderRepository.findById(orderId)
+                .orElseThrow(NoSuchElementException::new);
+        findOrder.changeOrderStatus(OrderStatus.valueOf(request.getOrderStatus()));
```

Comment:
```suggestion
        findOrder.changeOrderStatus(request.getOrderStatus());
```

Request에서 값을 처리해주는 경우도 있고, Service에서 해주는 경우도 있는데 각 선택의 기준이 있을까요?